File: GrocOrder/GrocOrder/FoodTypeService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GrocOrder
{
    public class FoodTypeService
    {
        public FoodType DetermineFoodType(string foodName)
        {
            FoodType ft = FoodType.FOOD;
            switch (foodName.ToUpper())
            {
                case "BURGER":
                case "BREAD":
                case "MILK":
                case "CEREAL":
                case "COFFEE":
                    ft = FoodType.FOOD;
                    break;
                case "LIGHT BULB":
                case "TOOTH PICK":
                case "CHARCOAL":
                case "PAPER TOWELS":
                    ft = FoodType.NONFOOD;
                    break;
                default:
                    break;
            }

            return ft;
        }
    }
}

File: eclipse/GrocOrder_ManualDI/src/demo/grocOrder/FoodTypeService.java
package demo.grocOrder;


public class FoodTypeService
{
    public FoodType DetermineFoodType(String name)
    {
        FoodType ft = FoodType.FOOD;
        String upper = name.toUpperCase();
        
        if ( upper.equals("BURGER") || upper.equals("BREAD")
                || upper.equals("MILK") || upper.equals("CEREAL")
                || upper.equals("COFFEE") )
        {
            ft = FoodType.FOOD;
        }
        else if (upper.equals("LIGHT BULB") || upper.equals("TOOTH PICKS")
                || upper.equals("CHARCOAL") || upper.equals("PAPER TOWELS") )
        {
            ft = FoodType.NONFOOD;
        }
        
        return ft;
    }
}
